src/index.ts

```diff
@@ -0,0 +1,87 @@
+import "zx/globals";
+import { parseArgs } from "node:util";
+import {
+  VERSION,
+  prepareConfigPath,
+  showHelp,
+  supportedBuildTypes,
+} from "./lib.js";
+
+import { login } from "./login.js";
+import { sync } from "./sync.js";
+import { build } from "./build.js";
+
+const commands = {
+  sync,
+  login,
+  build,
+};
+
+type Command = keyof typeof commands;
+
+export const main = async () => {
+  let args;
+  try {
+    args = parseArgs({
```

Comment:
Please colocate args and help in one module. Otherwise easy to forget to update one of them.